run_unittests.py

```diff
@@ -1138,6 +1139,28 @@ def test_syntax_highlighting_files(self):
             defined = set([a.strip() for a in res.group().split('\\')][1:])
             self.assertEqual(defined, set(chain(interp.funcs.keys(), interp.builtin.keys())))
 
+    def test_json_grammar_syntax_highlighting(self):
+        '''
+        Ensure that syntax highlighting JSON grammar written by TingPing was
+        updated for new functions in the global namespace in build files.
+        https://github.com/TingPing/language-meson/
+        '''
+        env = get_fake_env()
+        interp = Interpreter(FakeBuild(env), mock=True)
+        url = 'https://raw.githubusercontent.com/TingPing/language-meson/master/grammars/meson.json'
```

Comment:
This needs to be skipped when running in a non-networked build setting such as Flatpak and distro package building. Can't think of a good automatic way to detect that, though...

Reply:
I think it should be fine to skip the test if we get a 3xx status code?